src/app/src/mainwindow.cpp

```diff
@@ -168,6 +168,8 @@ void MainWindow::Initialize() {
 
     //initial camera setup, need this to allocate buffers
     m_camera->SetupExp(m_expSettings);
+    // SetupExp will set this value, so need to update here
+    m_expSettings.bufferCount = m_camera->ctx->curExp->bufferCount;
```

Comment:
The issue I was running into is that `SetupExp` sets bufferCount itself and that change doesn't get propagated back to `m_expSettings`, so then when `UpdateExp(m_expSettings)` gets called the bufferCount reverts back to the default value which for me was 0 and would cause acquisition to fail